README.rst

```diff
@@ -6,13 +6,166 @@ App for tracking sprint commitments, vacations and spillovers.
+    There has been a bug before that disallowed completing the sprint if it had archived issues, so we're moving all of them out of the active sprint.
+4. Close the active sprint.
+5. Move issues from the closed sprint to the next one.
+6. Open the next sprint.
+7. Create role-specific tasks for the future sprint.
+    The assignees for these tickets are retrieved from the `GOOGLE_ROTATIONS_RANGE` defined within `GOOGLE_ROTATIONS_SPREADSHEET`. The format of this document is the following:
+
+    a) First column contains sprint number (you can create multiple role tasks for one week by dividing sprint into parts, e.g. `Sprint 100a, Sprint 100b` - each in a separate row).
+    b) Next columns' headers contain role names prefixed by the full cell name (e.g. `Cell_1 FF`) and their fields contain assignees for the tickets.
+
+    The metadata (name, of these ticket is defined in `JIRA_CELL_ROLES`. Please see its docstring for the detailed explanation of its format.
+
+
+Sustainability
+^^^^^^^^^^^^^^
+Both dashboards are aware of the sprint board’s current view (whether it’s showing cells/cell’s board/person’s board). Therefore, when you click on the cell’s name, the sustainability dashboard recalculates its data for displaying cell/person-related data only.
+
+**TODO**: there is a rework of the columns planned for the next sprint, so this section will be expanded.
+
+Setting up budgets
+~~~~~~~~~~~~~~~~~~
+To set up the budgets for the accounts you need to:
+
+1. Log into the backend admin (by default it's http://localhost:8000/admin) with your superuser account.
+2. Go to `Sustainability/Budgets`.
+3. Add a new budget for the account.
+
+The budgets are rolling, so these entries are perceived as *changes* of the budgets. It means that the budget for the account with the specified `name` will be `hours` (per month) up to the next change or current date.
+
+    E.g. we have the account "Account - Security". From the beginning of 2019 we want the budget to be 100h/month, but from September to November (both inclusive) we want to raise it to 200h/month. From December and for the whole 2020 it should be lowered back to 100h/month. Therefore we need to create 3 entries via the Django admin:
+
+    .. code:: javascript
+
+        [{
+            "name": "Account - Security",
+            "date": January 2019,
+            "hours": 100
+        }, {
+            "name": "Account - Security",
+            "date": September 2019,
+            "hours": 200
+        }, {
+            "name": "Account - Security",
+            "date": December 2019,
+            "hours": 100
+        }]
+
+    Side note: the `date` is a `DateField`, but the example is using simplified representation for brevity.
+
+
+Sustainability Dashboard
+~~~~~~~~~~~~~~~~~~~~~~~~
+This view presents the assumptions described in `handbook's cell_budgets`_.
+The key information here is the ratio of non-billable cell responsible to billable hours. It is calculated in the following way:
```

Comment:
@Agrendalath Typos:

* Remove "responsible".
* Add "hours" to "non-billable cell".
* Add "cell" to "billable hours".

The final result would be:

> ... is the ratio of non-billable cell  hours to billable cell hours.